cfgov/cfgov/settings/base.py

```diff
@@ -259,3 +260,14 @@
 
 # PDFReactor
 PDFREACTOR_LIB = os.environ.get('PDFREACTOR_LIB', '/opt/PDFreactor/wrappers/python/lib')
+
+# S3 Configuration
+if os.environ.get('S3_ENABLED', False):
+    DEFAULT_FILE_STORAGE = 'v1.s3utils.MediaRootS3BotoStorage'
+    AWS_S3_SECURE_URLS = False  # True = use https; False = use http
+    AWS_QUERYSTRING_AUTH = False  # don't add complex authentication-related query parameters for requests
```

Comment:
Very minor, but this code comment could be updated to follow the format in L:267 for clarity. If at some point someone is in here.